Documented the non-normative use of PROV to resolve ticket #425 (#440)
w3c/sdw-sosa-ssn: ssn/rdf/examples/ACME-Beer-Example.md

product being consumed.  The ability to align SSN and product vocabularies
are beneficial in that it permits a unified description of the sensor.  This
includes modeling the sensor throughout its lifetime, announcing commercial
availability, procurement, deployment, operation, decommissioning and
disposal.

To this end this example models IBS TH2 sensors using [schema.org](https://schema.org/) Product and

> [GS1 WebVoc](https://ref.gs1.org/voc/) Product classes.  This permits concurrent support for both Search Engine Optimization (SOA) and consumer supply chain product identification using bar-codes.

Reviewer: The world seems to be split between "bar codes" and "barcodes"; hyphens are out. Our shortname is the one word "barcodes". Thus —
```suggestion
[GS1 WebVoc](https://ref.gs1.org/voc/) Product classes.  This permits concurrent support for both Search Engine Optimization (SOA) and consumer supply chain product identification using barcodes.
```

Section 9.6 of 
the [GS1 Digital Link
Standard](https://www.gs1.org/docs/Digital-Link/GS1_Digital_link_Standard_i1.1.1.pdf), 
GS1 deals with this same problem by sub-classing its [Product](https://ref.gs1.org/voc/Product)
class for both unique instances or indicating lot membership. Concurrently
supporting both vocabularies obviously requires that a physical instance of
a sensor be both an instance and a class. 

Note: This example makes concurrent use of the [Provenance Ontology](https://www.w3.org/TR/prov-o/) to model some of the operations in a
way that is incompatible with the [alignment module for the Provenance
Ontology]((https://www.w3.org/TR/vocab-ssn/#PROV_Alignment_property) provided by SOSA. Implementers are reminded that

> alignments and examples are both non-normative components of SOSA. Practitioners may be required to make modeling

Reviewer 2: ```suggestion
both alignments and examples are non-normative components of SOSA. Practitioners may be required to make modeling
```

> decisions that are appropriate to their application using their own alignments. The use of long identifiers is for readability purposes and is non-normative.

Reviewer: ```suggestion
decisions that are appropriate to their application using their own alignments. Long identifiers are used for readability purposes; this is non-normative.
```
